src/common/boards/nrf_PCA10156_1.0.0_L15.json

```diff
@@ -0,0 +1,125 @@
+{
+    "board": {
+        "boardVersion": "PCA10156",
+        "boardRevision": "1.0.0",
+        "boardName": "nRF54L15 PDK (rev. 1.0.0)"
+    },
+    "pins": [
+        {
+            "type": "vcom",
+            "id": "vcom0",
+            "name": "VCOM0",
+            "enable": {
+                "pin": 42
+            },
+            "hwfc": {
+                "pin": 20
+            }
+        },
+        {
+            "type": "vcom",
+            "id": "vcom1",
+            "name": "VCOM1",
+            "enable": {
+                "pin": 22
+            },
+            "hwfc": {
+                "pin": 23
+            }
+        },
+        {
+            "type": "switch",
+            "id": "ledcontrol",
+            "title": "Power to LEDs",
+            "label": "Disable to have LEDs unpowered (and kept off).",
+            "tooltip": "Enabling this option provides LEDs from LED0 to LED3 with 3.3 V of power.",
+            "enable": {
+                "pin": 45
+            }
+        },
+        {
+            "type": "switch",
+            "id": "swd-control",
+            "title": "Software Debugger (SWD)",
+            "label": "Disable to use an external debugger on the target chip.",
+            "tooltip": "When enabled, the SWD is connected and the IMCU is acting as the debugger. When disabled, the SWD is disconnected and you can use an external debugger connected to the Debug In header.",
+            "enable": {
+                "pin": 6
+            }
+        },
+        {
+            "type": "switch",
+            "id": "swo-control",
+            "title": "SWO Control",
+            "label": "Connect or disconnect the SWO signal",
+            "tooltip": "When enabled, connects the SWO signal connection to the SWO P2.07 pin on nRF54L15. When disabled, the SWO signal is disconnected.",
+            "enable": {
+                "pin": 9,
+                "invert": true
+            }
+        },
```

Comment:
second new pin controls to review, based on LV10 description. Core difference is that "Active Low" -> thats why inverted.